Library/Formula/xalan-c.rb

```diff
@@ -30,6 +35,17 @@ def install
                             "--disable-silent-rules",
                             "--prefix=#{prefix}"
       system "make", "install"
+      if build.with? "docs"
+        ENV.prepend_path "PATH", "#{buildpath}/c/bin"
+        cd "xdocs" do
+          # Set the library path in the script which runs Xalan from
+          # the source tree, or else the libxalan-c.dylib won't be found.
+          inreplace "sources/make-xalan.sh", "\"${XALANCMD}\" \\",
+                    "export DYLD_FALLBACK_LIBRARY_PATH=#{buildpath}/c/lib:$DYLD_FALLBACK_LIBRARY_PATH\n\"${XALANCMD}\" \\"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!